src/lib/structures/Command.ts

```diff
@@ -484,6 +492,13 @@ export interface CommandOptions extends AliasPieceOptions, FlagStrategyOptions {
 	 * @default null
 	 */
 	runIn?: CommandOptionsRunType | CommandOptionsRunTypeEnum | readonly (CommandOptionsRunType | CommandOptionsRunTypeEnum)[] | null;
+
+	/**
+	 * If the client's options have set typing as true, setting typing as false in the command's options will override it.
+	 * Setting it to true will have no efect, you can call `message.channel.sendTyping()` on the run method if you only want specific commands to display the typing status.
```

Comment:
```suggestion
	 * If {@link SapphireClient.typing} is true, this option will override it.
	 * Otherwise, this option has no effect - you may call {@link Channel#sendTyping}` in the run method if you want specific commands to display the typing status.
```

I'm not entirely sure how linking to `discord.js` elements is handled in Sapphire's documentation - replace `Channel#sendTyping` as necessary.